wdl/transforms/cascades/src/test/scala/wdl/transforms/cascades/linking/expression/types/CascadesTypeEvaluatorSpec.scala

```diff
@@ -64,4 +64,12 @@ class CascadesTypeEvaluatorSpec extends AnyFlatSpec with CromwellTimeoutSpec wit
       e.evaluateType(Map.empty) shouldBeValid WomArrayType(WomStringType)
     }
   }
+
```

Comment:
There's another test in the Biscayne version, should it be added here?

Reply:
I think we're still missing a test here that is is Biscayne, no? Once that's in I think this is good to go!